Add WebGL and custom render API with TresJS ways-of-using-vue.md (#2651)
vuejs/docs: src/guide/extras/ways-of-using-vue.md

- Build desktop apps with [Electron](https://www.electronjs.org/) or [Tauri](https://tauri.app)
- Build mobile apps with [Ionic Vue](https://ionicframework.com/docs/vue/overview)
- Build desktop and mobile apps from the same codebase with [Quasar](https://quasar.dev/)

> - Build 3D WebGL Experiences with [TresJS](https://tresjs.org/)
> - Use Vue's [Custom Renderer API](/api/custom-renderer) to build custom renderers targeting [WebGL](https://tresjs.org/) or even [the terminal](https://github.com/vue-terminal/vue-termui)!

Reviewer: ```suggestion
- Build 3D WebGL experiences with [TresJS](https://tresjs.org/)
- Use Vue's [Custom Renderer API](/api/custom-renderer) to build custom renderers, like those for [the terminal](https://github.com/vue-terminal/vue-termui)!
```

Author: Hey there, thanks for the quick feedback, let's go for the first option to highlight that is more than just a custom render but an ecosystem as such and remove it from the last point.
